Copy edit 5.16 guides (#2010)
sensu/sensu-docs: content/sensu-go/5.16/guides/securing-sensu.md

etcd-peer-trusted-ca-file: "/path/to/your/peer/ca/file"
{{< /highlight >}}

## Secure the API and dashboard

> Both the Sensu Go API and the dashboard use a common stanza in `/etc/sensu/backend.yml` to provide the certificate, key, and CA file needed to provide secure communication.

Reviewer: ```suggestion
The Sensu Go Agent API, HTTP API, and web UI use a common stanza in `/etc/sensu/backend.yml` to provide the certificate, key, and CA file needed to provide secure communication.
```

insecure-skip-tls-verify: false
{{< /highlight >}}

> Providing these cert-file and key-file parameters will cause the API to serve HTTP requests over SSL/TLS (https).

Reviewer: ```suggestion
Providing these cert-file and key-file parameters will cause the Agent Websocket API and HTTP API to serve requests over SSL/TLS (https).
```

trusted-ca-file: "/path/to/ca.pem"
{{< /highlight >}}

You can use use certificates for authentication that are distinct from other communication channels used by Sensu, like etcd or the API.
However, deployments can also use the same certificates and keys for etcd peer and client communication, the HTTP API, and agent authentication without issues.

## Create self-signed certificates for securing etcd and backend-agent communication {#create-self-signed-certificates}

This example uses the [cfssl][9] tool to generate self-signed certificates.

First, create a Certificate Authority (CA).

> To keep things straightforward in this example, you will generate all clients and peer certificates using this CA, but you might eventually want to create distinct CA.

Reviewer: ```suggestion
To keep things straightforward in this example, you will generate all clients and peer certificates using this CA, but you might eventually want to create a distinct CA.
```
